integration/spark/README.md

```diff
@@ -4,22 +4,44 @@ The OpenLineage Spark Agent uses jvm instrumentation to emit OpenLineage metadat
 
 ## Installation
 
+### Version 1.8.0 or earlier
+
 Maven:
 
 ```xml
 <dependency>
     <groupId>io.openlineage</groupId>
     <artifactId>openlineage-spark</artifactId>
-    <version>1.8.0</version>
+    <version>${OPEN_LINEAGE_VERSION}</version>
+</dependency>
+```
+
+or Gradle:
+
+```groovy
+implementation("io.openlineage:openlineage-spark:${OPEN_LINEAGE_VERSION}")
+```
+
+### Version 1.9.0 or later
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>io.openlineage</groupId>
+    <artifactId>openlineage-spark_${SCALA_BINARY_VERSION}</artifactId>
```

Comment:
I think this needs additional information on https://github.com/OpenLineage/docs/ and probably website/slack.

Unless, we might still want to release `openlineage-spark` as 2.12 version for compatibility for a few versions?

Reply:
I'd rather take the "communicate" approach, instead of the "let's add complexity to our build" approach.

What do you think?